frontend/projects/upgrade/src/app/features/dashboard/feature-flags/modals/delete-feature-flag-modal/delete-feature-flag-modal.component.ts

```diff
@@ -44,9 +44,7 @@ export class DeleteFeatureFlagModalComponent {
   private inputSubject: BehaviorSubject<string> = new BehaviorSubject<string>('');
 
   // Observable that emits true if inputValue is 'delete', false otherwise
-  isDeleteNotTyped$: Observable<boolean> = this.inputSubject
-    .asObservable()
-    .pipe(map((value) => value.toLowerCase() !== 'delete'));
+  isDeleteNotTyped$: Observable<boolean> = this.inputSubject.pipe(map((value) => value.toLowerCase() !== 'delete'));
```

Comment:
this has messed up something, `isDeleteActionBtnDisabled` is always false and I can delete the feature flag without typing anything. Please verify code working fine after the code change.